packages/traefik/spec.yml

```diff
@@ -32,55 +31,26 @@ targets:
   - src: '{{.SpecRoot}}/{{.Name}}.service'
     dest: /etc/systemd/system/{{.Name}}.service
     template: yes
-  - src: '{{empty}}'
-    dest: /etc/traefik
-  - src: '{{empty}}'
-    dest: /etc/traefik/configs
-
-  # logging
-  - src: '{{.SpecRoot}}/logrotate/traefik'
-    dest: /etc/logrotate.d/traefik
-  - src: '{{.SpecRoot}}/logrotate/traefik-access'
-    dest: /etc/logrotate.d/traefik-access
-  - src: "{{empty}}"
-    dest: /var/log/{{.Name}}
-  - src: "{{empty}}"
-    dest: /var/log/{{.Name}}/archive
-
-  # config generation scripts
-  - src: '{{.SpecRoot}}/config/'
-    dest: /usr/bin/
-  - src: '{{.SpecRoot}}/traefik-restart'
-    dest: /etc/sudoers.d/
 
 scripts:
-  before-install: |
-    getent group {{.Name}} > /dev/null || groupadd -r {{.Name}}
-    getent passwd {{.Name}} > /dev/null || \
-        useradd -r \
-                -g {{.Name}} \
-                -d /etc/{{.Name}} \
-                -s /sbin/nologin \
-                -c "{{.Name}} user" \
-                {{.Name}}
-
   after-install: |
-    chown -R {{.Name}}:{{.Name}} /etc/{{.Name}}
+    chown -R root:root /etc/{{.Name}}
```

Comment:
@BrianHicks Is here -- https://github.com/asteris-llc/mantl-packaging/pull/134/files/95fd6cf114b34d672bc1d0cf56b4d41810ea409e#diff-189888afe84055bbd5a83450b1a6be8dR43 just moved to config package, this part should be shared with cisco stuff